lua/plenary/path.lua

```diff
@@ -19,6 +19,24 @@ local S_IF = {
 local path = {}
 path.home = vim.loop.os_homedir()
 
+path.root = (function()
```

Comment:
Can't we move this below `path.sep` and use `path.sep` to get the `root` rather than doing the `if jit else etc` twice